README.md

```diff
@@ -2,3 +2,30 @@ FuseSoC standard core library
 =============================
 
 This is the standard core library to be used with [FuseSoC](https://github.com/olofk/fusesoc)
+
+### Installation
+
+Clone this repository and add it to your `fusesoc.conf`.  For example
+in `~/.config/fusesoc/fusesoc.conf`:
+
+```
+[main]
+cores_root =
+ /home/joe/work/fusesoc-cores
+cache_root = /home/joe/work/fuse-cache
+build_root = /home/joe/work/fuse-builds
+```
+
+### Contributing
+
+Cores for fusesoc should follow the following guidelines.
+
+ - **No Code** Please do not store code in `fusesoc-cores`.  Please store
+   your code in a separate repo or somehwere it can be fetch with the `url`
+   provider.
+ - If your core is really small please consider storing in the
+   [tiny-cores](http://github.com/fusesoc/tiny-cores) repo.
+ - **Stay Modern** Use the modern sections in your core file like `fileset`
```

Comment:
Refer to the [migration guide](https://github.com/olofk/fusesoc/blob/master/doc/migrations.adoc)

Reply:
Yeah, I was planning to add that after you referred me to that.